rpcm/dem.py

```diff
@@ -0,0 +1,9 @@
+import rasterio as rio
+
+GCS_COPERNICUS_DEM_VRT = "gs://overstory-dtms/CopernicusDem30m/copernicus_dem/dem.vrt"
+
+def get_copernicus_elevation(lon: float, lat: float):
+    with rio.open(GCS_COPERNICUS_DEM_VRT) as src:
+        assert src.crs == "EPSG:4326"
+        elevation = list(src.sample([(lon, lat)]))[0]
+    return elevation
```

Comment:
I'm not sure rasterio can read VRT files on GCS like this. Need to test first.

Reply:
Seems like it should be possible: https://kokoalberti.com/articles/hosting-and-accessing-cloud-optimized-geotiff-files-on-aws-s3/